plugins/nordic-hid/fu-nordic-hid-cfg-channel.c

```diff
@@ -916,6 +916,12 @@ fu_nordic_hid_cfg_channel_get_hwid(FuNordicHidCfgChannel *self, GError **error)
 				      hw_id[7]);
 	fu_device_set_physical_id(FU_DEVICE(self), physical_id);
 
+	/* Devices connected via a dongle use physical ID as name to avoid inheriting name from the
+	 * dongle.
+	 */
+	if (self->peer_id != 0)
```

Comment:
can we use a `#define` for this please? e.g. `FU_NORDIC_HID_PEER_ID_RECEIVER` or something like that?

Reply:
Added `PEER_ID_LOCAL_RECIPIENT` globally (as a separate commit)